specification/2.0/README.md

```diff
@@ -1250,6 +1250,8 @@ glTF 2.0 also supports animation of instantiated Morph Targets in a similar fash
 
 > **Note:** glTF 2.0 defines only animation storage, so this specification doesn't define any particular runtime behavior, such as: order of playing, auto-start, loops, mapping of timelines, etc...
 
+> **Note:** glTF 2.0 does not specifically define how an animation will be used when imported but, as a best practice, it is recommended that each animation is self contained as an action. This clarification is necessary due to early glTF 2.0 exporters splitting the tracks of a single source animation into several glTF animations (behaviour that is not currently advised today).
```

Comment:
Couple things:

* [ ] For consistency with the rest of the document, I would title this paragraph: **Implementation Note:**
* [ ] IMO the second sentence, giving historical background about a (hopefully soon-to-be-fixed) bug in the sample models, can be omitted from the specification.
* [ ] Would suggest additional language explaining the concept of an action: *"For example, "Walk" and "Run" animations might each contain multiple channels targeting a model's various bones. The viewer or engine may choose when to play any of the available animations."*